### ISSUE TITLE
macOS build failure for 2.6.0

### ISSUE THREAD

**Issue by ilovezfs** [Qalculate/libqalculate#96]
Full log https://gist.github.com/ilovezfs/90800478d2688336005632b442e077e0

The error is
```
/bin/sh ../libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I..  -I.. -DPACKAGE_DATA_DIR=\""/usr/local/Cellar/libqalculate/2.6.0/share"\" -I/usr/include/libxml2     -g -O2 -c -o util.lo util.cc
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c util.cc  -fno-common -DPIC -o .libs/util.o
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c util.cc -o util.o >/dev/null 2>&1
/bin/sh ../libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I..  -I.. -DPACKAGE_DATA_DIR=\""/usr/local/Cellar/libqalculate/2.6.0/share"\" -I/usr/include/libxml2     -g -O2 -c -o Unit.lo Unit.cc
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c Unit.cc  -fno-common -DPIC -o .libs/Unit.o
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c Calculator.cc -o Calculator.o >/dev/null 2>&1
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c Unit.cc -o Unit.o >/dev/null 2>&1
/bin/sh ../libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I..  -I.. -DPACKAGE_DATA_DIR=\""/usr/local/Cellar/libqalculate/2.6.0/share"\" -I/usr/include/libxml2     -g -O2 -c -o BuiltinFunctions.lo BuiltinFunctions.cc
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c BuiltinFunctions.cc  -fno-common -DPIC -o .libs/BuiltinFunctions.o
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c MathStructure.cc -o MathStructure.o >/dev/null 2>&1
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c BuiltinFunctions.cc -o BuiltinFunctions.o >/dev/null 2>&1
/bin/sh ../libtool  --tag=CXX   --mode=compile clang++ -DHAVE_CONFIG_H -I. -I..  -I.. -DPACKAGE_DATA_DIR=\""/usr/local/Cellar/libqalculate/2.6.0/share"\" -I/usr/include/libxml2     -g -O2 -c -o QalculateDateTime.lo QalculateDateTime.cc
libtool: compile:  clang++ -DHAVE_CONFIG_H -I. -I.. -I.. -DPACKAGE_DATA_DIR=\"/usr/local/Cellar/libqalculate/2.6.0/share\" -I/usr/include/libxml2 -g -O2 -c QalculateDateTime.cc  -fno-common -DPIC -o .libs/QalculateDateTime.o
QalculateDateTime.cc:1553:11: error: definition of variable with array type needs an explicit size or an initializer
        long int coefficients[] {
                 ^
QalculateDateTime.cc:1553:25: error: expected ';' at end of declaration
        long int coefficients[] {
                               ^
                               ;
QalculateDateTime.cc:1840:11: error: definition of variable with array type needs an explicit size or an initializer
        long int sine_coeff[] {
                 ^
QalculateDateTime.cc:1840:23: error: expected ';' at end of declaration
        long int sine_coeff[] {
                             ^
                             ;
4 errors generated.
make[1]: *** [QalculateDateTime.lo] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [install-recursive] Error 1
```

**Comment by hanna-kn** [comment]
Should be fixed by commit d1e6bbe

**Comment by ilovezfs** [comment]
Awesome! Thanks @hanna-kn. Do you want me to backport that for Homebrew or wait on a new tag?

**Comment by hanna-kn** [comment]
By the way, I noticed that CLN, Glib and Wget are listed is dependencies. They are not used any more (CLN was replaced GMP/MPFR and Wget by curl.

**Comment by ilovezfs** [comment]
Thanks @hanna-kn. I've removed those dependencies in https://github.com/Homebrew/homebrew-core/pull/28911 and shipped 2.6.0 in Homebrew.

**Comment by hanna-kn** [comment]
I discovered another small mistake so I decided to create a new tarball (which can be used without the patch) with both fixes:
https://github.com/Qalculate/libqalculate/releases/download/v2.6.0a/libqalculate-2.6.0b.tar.gz
